src/Layer/babylon.highlightlayer.ts

```diff
@@ -826,7 +826,7 @@
                 }
             }
 
-            this._meshes[mesh.uniqueId] = null;
+            delete this._meshes[mesh.uniqueId];

```

Comment:
just move it two lines before to be sure that this._meshes[mesh.uniqueId] exists

Reply:
This actually works cause the highlight layer code expect to have null values in the list but agree it might look neater without them.